data/hai/hai news.txt

```diff
@@ -204,6 +250,7 @@ news "concerned hai in Hai space"
 			"It would be smart to follow trade routes that only visit Hai and human space. One would not wish to experience the business of our northern brethren."
 			"Seeing so many human ships come by with such extensive weaponry feels very reminiscent of past times, unfortunately."
 			"Rations are very often sent to those starving to the north of here, but such actions are almost never met with similar kindness. Take such a task very carefully."
+			"I feel so terrible for the many humans that attempt to reach our worlds and are destroyed by the Unfettered humans that block their passage."
```

Comment:
Aren't unfettered humans Alphas? I don't think Alphas are hanging around the Hai wormhole.

Reply:
In one of the missions where pirates go through the wormhole, the Hai refer to them as 'Unfettered humans', so I felt it was appropriate to refer to them the same here.

Reply:
Ok then, feel free to resolve.